node/src/BaseClient.ts

```diff
@@ -571,6 +574,17 @@ export interface BaseClientConfiguration {
      * used.
      */
     inflightRequestsLimit?: number;
+    /**
+     * Availability Zone of the client.
+     * This setting ensures that read operations are directed to nodes within the specified AZ.
+     * If not set, the client's AZ will be determined dynamically based on the node addresses or default behavior.
+     *
+     * @example
+     * ```typescript
+     * configuration.client_az = 'us-east-1a';
```

Comment:
Does this is the only type possible? should we restrict to one working standard or making sure we tell the user he was wrong?

Reply:
wdym here? enforcing the client_az to be in a certain format? and if not, throw an error message?

Reply:
Not throwing, loging.
Its more for the core level as its mutual, but the question is do we know what should be the right form and help the user more in order to not make mistakes.
Even on the level of saying `known optional forms: AWS EC2 uses ... AWS EKS uses ... GCP uses ...` so at least its easy for him to validate it.

Reply:
example is incomplete without setting proper read from strategy. It is also kinda obvious, no?

Reply:
Everyone has their own technical area and things can be confusing when you meet them first time.
Somebody might think its `1a` for example.